examples/send-and-receive-emails/send-and-receive-emails-2-pop.bal

```diff
@@ -0,0 +1,28 @@
+import ballerina/email;
+import ballerina/io;
+
+public function main() {
+    // Create the client with the connection parameters, host, username, and password. An error is received in failure.
+    email:PopClient|email:Error popClient = new ("pop.email.com", "reader@email.com", "pass456");
+    // Check whether the email is received successfully. Otherwise, print the error message.
```

Comment:
This will check the client initialization not the email received. Shall we update the sentence here?

Reply:
I think we can get rid of this sentence altogether. We should only be commenting code relevant to the concepts being demonstrated by the example.

Reply:
Went through both examples, and most of the comments are actually unnecessary (i.e., doesn't add anything more than what you can understand by reading the code)

Reply:
@wggihan Corrected.

@pubudu91 Went through the comments again and changed accordingly.